packages/core/src/components/group/group.scss

```
@@ -145,28 +166,50 @@ $header-height-focus-visible: calc($header-height - 2px);
-    color: var(--theme-color-std-text);
+    color: var(--ix-group-expand-icon-color);
   }
 
   .btn-expand-header {
+    appearance: none;
+    box-sizing: border-box;
+    display: flex;
+    flex: 0 0 auto;
+    align-items: flex-start;
+    justify-content: center;
+    // Native <button> flex items do not stretch like the old <div>; pin to top.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)